src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Ball;
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_NUMBER = 100;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random randomNumber = new Random();
```

Comment:
it's not random number but random generator, better just name it as `random`

Reply:
Ok